### ISSUE TITLE
About label_len

### ISSUE THREAD

**Issue by yzz9048** [SJTU-DMTai/OnlineTSF#6]
pardon again, please
for forecasting task, here is a parameter named "label_len". What is the function of this parameter? 
In your implementation of loading data, it seems that seq_y contains the data of the final label_len length of seq_x. However, this process seems not to be reflected in Definitions and Problem of your  paper.

**Comment by yzz9048** [comment]
Additionally, when I debug the train process of the forecasting task for ETTh dataset, the self.recent_batch in line https://github.com/SJTU-DMTai/OnlineTSF/blob/466b682e08ce1bd676f783d7fa74efae21651258/adapter/proceed.py#L53 is a zero tensor, which may stem form self.register_buffer('recent_batch', torch.zeros(1, args.seq_len + args.pred_len, args.enc_in), persistent=False) in __init__ of class Proceed. So it cannot represent the extraction of the most recent batch mentioned in the paper, i.e. D_{t-}. Did I make a mistake somewhere, or did I fail to understand something correctly？

**Comment by MogicianXD** [comment]
> a parameter named "label_len". What is the function of this parameter? 

It is never used in PatchTST and iTransformer. The code originates from [Informer](https://github.com/zhouhaoyi/Informer2020) models and some follow-up works.

**Comment by MogicianXD** [comment]
> Additionally, when I debug the train process of the forecasting task for ETTh dataset, the self.recent_batch is a zero tensor, which may stem form self.register_buffer('recent_batch', torch.zeros(1, args.seq_len + args.pred_len, args.enc_in), persistent=False) in **init** of class Proceed. So it cannot represent the extraction of the most recent batch mentioned in the paper, i.e. D_{t-}. Did I make a mistake somewhere, or did I fail to understand something correctly？

It would be somewhat troublesome to always feed labels into the `forward` of our model. Thus, we assign the recent batch to the model in the outer space [here](https://github.com/SJTU-DMTai/OnlineTSF/blob/466b682e08ce1bd676f783d7fa74efae21651258/exp/exp_proceed.py#L96), though sacrificing readability.

**Comment by MogicianXD** [comment]
Thanks for your attention to our work! Nevertheless, for higher performance in practice, we would recommend combining our online learning methods with [replaying](https://github.com/SJTU-DMTai/OnlineTSF/blob/466b682e08ce1bd676f783d7fa74efae21651258/exp/exp_online.py#L349) techniques during the online phase, instead of updating the model with only one sample. We have not supported this feature yet.

**Comment by yzz9048** [comment]
Thanks, I see.
